wiki/src/img/google.png

```diff
@@ -0,0 +1,7 @@
+<html>
```

Comment:
Umm, I don't think this is a png!

Reply:
https://github.com/wetfish/classic/pull/38/commits/628078c0acbb29b9d4ad5365ee420d90ca2cff23 should be png now